crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/controller/CrimeApplicationController.java

```diff
@@ -51,7 +52,12 @@ public class CrimeApplicationController {
     )
     public MaatApplication getCrimeApplyData(@PathVariable Long id) {
         log.info("Get applicant details from Crime Apply datastore");
-        eformStagingService.retriveOrInsertDummyUsnRecord(id);
-        return crimeApplicationService.retrieveApplicationDetailsFromCrimeApplyDatastore(id);
+        EformStagingResponse eformStagingResponse = eformStagingService.retriveOrInsertDummyUsnRecord(id);
+        MaatApplication maatApplication = crimeApplicationService.retrieveApplicationDetailsFromCrimeApplyDatastore(id);
+        if(eformStagingResponse.getMaatRef() != null) {
```

Comment:
I would consider extracting a variable here to reduce duplication of `eformStagingResponse.getMaatRef()` and we also now have an opportunity to give this new variable a name and add some description to what it is, see example below.

```
        Integer amendedMaatRef = eformStagingResponse.getMaatRef();
        if (amendedMaatRef != null) {
            return maatApplication
                    .withMaatRef(amendedMaatRef);
        }
```

Reply:
eformStagingResponse.getMaatRef() can be null. If I extract to a variable then I need an inline ternary null check like: 
       Integer maatRef = eformStagingResponse.getMaatRef() != null ? eformStagingResponse.getMaatRef() : null ;
        if(maatRef != null) {
            return maatApplication
                    .withMaatRef(maatRef);
        }
It doesn't reduce the duplication.

Reply:
Sorry, it will not throw NullPointer, it is a null value. I will extract it to a variable.

Reply:
Change looks good